packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,512 @@
+        isRowHeader: true,
+        isResizable: true,
+        onRender: this.createRenderCellInterface('name'),
+        columnActionsMode: ColumnActionsMode.disabled
+      },
+      {
+        key: 'signature',
+        name: 'Signature',
+        fieldName: 'signature',
+        minWidth: 200,
+        maxWidth: 300,
+        isCollapsible: false,
+        isResizable: true,
+        isMultiline: true,
+        onRender: this.createRenderCellSignature('typeTokens'),
+        columnActionsMode: ColumnActionsMode.disabled
+      },
+      {
+        key: 'description',
+        name: 'Description',
+        fieldName: 'description',
+        minWidth: 300,
+        maxWidth: 400,
+        isCollapsible: false,
+        isResizable: true,
+        isMultiline: true,
+        onRender: this.createRenderCellInterface('description'),
+        columnActionsMode: ColumnActionsMode.disabled
+      }
+    ];
+
+    this._enumColumns = [
+      {
+        key: 'name',
+        name: 'Name',
+        fieldName: 'name',
+        minWidth: 150,
+        maxWidth: 250,
+        isCollapsible: false,
+        isRowHeader: true,
+        isResizable: true,
+        onRender: this.createRenderCellEnum('name')
+      },
+      {
+        key: 'value',
+        name: 'Value',
+        fieldName: 'value',
+        minWidth: 100,
+        maxWidth: 200,
+        isCollapsible: false,
+        isResizable: true,
+        onRender: this.createRenderCellEnum('value')
+      },
+      {
+        key: 'description',
+        name: 'Description',
+        fieldName: 'description',
+        minWidth: 300,
+        maxWidth: 400,
+        isCollapsible: false,
+        isResizable: true,
+        onRender: this.createRenderCellEnum('description')
+      }
+    ];
+  }
+
+  public render(): JSX.Element | null {
+    const { description, extendsTokens } = this.props;
+    const { properties, isEnum, isClass } = this.state;
+
+    return (
+      <Stack gap={MEDIUM_GAP_SIZE}>
+        {description && extendsTokens && extendsTokens.length > 0 ? (
+          <Stack gap={SMALL_GAP_SIZE}>
+            {this._renderTitle()}
+            {(description || (extendsTokens && extendsTokens.length > 0)) && (
+              <Stack gap={XSMALL_GAP_SIZE}>
+                {this._renderDescription()}
+                {this._renderExtends()}
+              </Stack>
+            )}
+          </Stack>
+        ) : (
+          this._renderTitle()
+        )}
+        {isClass
+          ? this._renderClass()
+          : properties.length >= 1 && (
+              <DetailsList
+                selectionMode={SelectionMode.none}
+                layoutMode={DetailsListLayoutMode.justified}
+                items={properties}
+                columns={isEnum ? this._enumColumns : this._defaultColumns}
+                onRenderRow={this._onRenderRow}
+              />
+            )}
+      </Stack>
+    );
+  }
+
+  private renderCell = (text: string, deprecated: boolean = false) => {
```

Comment:
nit, private methods are typically prefixed with _ (we should probably consider a lint rule enforcing this)